swift-ci/master/centos/7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
+
+RUN groupadd -g 42 build-user && \
+    useradd -m -r -u 42 -g build-user build-user
+
+RUN yum install -y epel-release centos-release-scl
+
+RUN yum install --enablerepo=centosplus  -y \
+	autoconf          \
+	cmake3            \
+	devtoolset-8      \
+	gettext           \
+	gnutls-devel      \
+	krb5-devel        \
+	libatomic         \
+	libedit-devel     \
+	libidn2-devel     \
+	libmetalink-devel \
+	libnghttp2-devel  \
+	libssh2-devel     \
+	libstdc++-devel   \
+	libtool           \
+	libuuid-devel     \
+	libxml2-devel     \
+	llvm-toolset-7    \
+	make              \
+	ncurses-devel     \
+	ninja-build       \
+	openldap-devel    \
+	openssl-devel     \
+	pexpect           \
+	python-devel      \
+	python-pygments   \
+	python36-pexpect  \
+	PyYAML            \
+	rpm-build.x86_64  \
+	rsync             \
+	sclo-git25-git    \
+	sqlite-devel      \
+	swig3             \
+	which             \
+	zlib-devel
```

Comment:
@discogestalt this list is significantly longer than what we do for Centos8 and other distributions. Is it the bare minimum to get Swift build & qualified?

Reply:
For the most part, yeah - let me confirm, there may be a package or two that were there for building libcurl (when that was necessary).